envoy/filter/config_provider_manager.h

```diff
@@ -23,7 +23,7 @@ using DynamicFilterConfigProviderPtr = std::unique_ptr<DynamicFilterConfigProvid
  * The FilterConfigProviderManager exposes the ability to get an FilterConfigProvider
  * for both static and dynamic filter config providers.
  */
-template <class FactoryCb> class FilterConfigProviderManager {
+template <class FactoryCb, class FactoryCt> class FilterConfigProviderManager {
```

Comment:
Yeah, I had thought about this. The reason I chose FactoryCt is because FactoryContext is a well defined class here: https://github.com/envoyproxy/envoy/blob/575cab42ec231c6e7dd5b5bad72a6ba4820e34fb/envoy/server/factory_context.h#L187
I had concern using it as a template type may cause confusion, like <class int32>.    Is FactoryCtext better?

Reply:
I'd suggest `FactoryCtx` since `Ct` may mean "constructor".

Reply:
Cool! ACK